project-keeper/src/main/java/com/exasol/projectkeeper/dependencyupdate/VulnerabilityInfoProvider.java

```diff
@@ -1,9 +1,53 @@
 package com.exasol.projectkeeper.dependencyupdate;
```

Comment:
What a wonderful class to read. No questions left open!

Reply:
🙇